### PR TITLE
DOC: improvement in scipy.interpolate.splprep

### PR DESCRIPTION
**Pull request by ashwinpathak20**
Please look at my changes applied in accordance with the issue #6618 in  scipy.interpolate.splprep.

### CODE REVIEW

File: scipy/interpolate/fitpack.py [hunk 1]
```diff
@@ -110,6 +110,7 @@ def splprep(x, w=None, u=None, ub=None, ue=None, k=3, task=0, s=None, t=None,
     -----
     See `splev` for evaluation of the spline and its derivatives.
     The number of dimensions N must be smaller than 11.
+    Splprep pads zeros at the end which is a cross-link to BSpline as BSpline does not require zero-padding.
```
Comment (ev-br) [inline scipy/interpolate/fitpack.py]: I don't think this sentence clarifies much.

Comment (ashwinpathak20) [inline scipy/interpolate/fitpack.py]: Then what should be added , I mean what should I focus upon ?

File: scipy/interpolate/fitpack.py [hunk 2]
```diff
@@ -110,6 +110,8 @@ def splprep(x, w=None, u=None, ub=None, ue=None, k=3, task=0, s=None, t=None,
     -----
     See `splev` for evaluation of the spline and its derivatives.
     The number of dimensions N must be smaller than 11.
+    The c array retruned consists of k+1 zeros at the end
+    while BSpline does not require zero-padding.
```
Comment (ev-br) [inline scipy/interpolate/fitpack.py]: Could be useful to contrast it to `splrep` (does it zero-pad?)

Comment (ashwinpathak20) [inline scipy/interpolate/fitpack.py]: Ok, so should I include the that splrep does not zero-pad also in the docstring?

**Comment by ev-br** [comment]
Thank you @ashwinpathak20 , your commits are squashed and merged in gh-7678. Your authorship is preserved :-)

**Pull request closed without merging**